drivers/QSPI.cpp

```diff
@@ -0,0 +1,409 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 Nordic Semiconductor ASA
```

Comment:
@sg-, @screamerbg, I'm still new to licensing things, is this copyright notice correct for mbed OS?